webapp/graphite/render/functions.py

```diff
@@ -3257,7 +3261,7 @@ def sortByMinima(requestContext, seriesList):
     &target=sortByMinima(server*.instance*.memory.free)
 
   """
-  newSeries = [series for series in seriesList if safeMax(series) > 0]
+  newSeries = [series for series in seriesList if safeMax(series, default=0) > 0]
```

Comment:
This doesn't seem like it would actually change the behavior of the function

Reply:
It does, because previously `safeMax()` might return None, and `None > 0` is a TypeError in Python 3.

Reply:
Ah ok, I didn't realize that was a problem on 3.x